x-pack/plugins/fleet/server/telemetry/sender.ts

```diff
@@ -168,7 +168,7 @@ export class TelemetryEventsSender {
       const resp = await axios.post(telemetryUrl, ndjson, {
         headers: {
           'Content-Type': 'application/x-ndjson',
-          'X-Elastic-Cluster-ID': clusterUuid,
+          ...(clusterUuid ? { 'X-Elastic-Cluster-ID': clusterUuid } : undefined),
```

Comment:
__note:__ `undefined` __values__ are no longer supported by the Axios headers types.